docs/notebooks/3.2 Comparing to Measurements - Quality Controlling and Gap Filling Measurements with Solcast Actuals.ipynb

```diff
@@ -0,0 +1,689 @@
+      "text/plain": [
+       "<Figure size 640x480 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "df = solcast.add_suffix(\"_solcast\").join(measurements.add_suffix(\"_meas\"))\n",
```

Comment:
.join(lsuffix="_solcast", rsuffix="_meas")

Reply:
actually different in this circumstance, because only collisions apply lsuffix, rsuffix